### DISCUSSION TITLE
Warning when updating via composer if a plugins license is expired.

### DISCUSSION THREAD

**Discussion by outline4** [craftcms/cms#9120] [Ideas] [answered]
It would be nice if I get a warning in composer when trying to update a plugin that has its license expired. 

Maybe ask a question like "the license of plugin x/y is expired: do you still want to continue (y/n)?",     
or don't allow updating it at all.

**Comment by brandonkelly** [comment, marked as answer]
It’s not possible to do this from normal `composer` commands, but Craft does factor license status into its own `update` command: https://craftcms.com/docs/3.x/updating.html#updating-from-the-terminal

**Reply by outline4** [reply]
Ok thanks. This is good to know.
